table/block_based_table_builder.cc

```diff
@@ -579,6 +579,8 @@ void BlockBasedTableBuilder::WriteBlock(const Slice& raw_block_contents,
     MeasureTime(r->ioptions.statistics, BYTES_COMPRESSED,
                 raw_block_contents.size());
     RecordTick(r->ioptions.statistics, NUMBER_BLOCK_COMPRESSED);
+  } else if (type != r->compression_ctx.type()) {
```

Comment:
There is no member ```type()``` in ```CompressionContext```. Is something missing here? This is failing compilation in our internal builds.

Reply:
https://github.com/facebook/rocksdb/commit/2c146622138165c62a186a59a54ec24f30d044b7 changed
```  r->compression_type;```
to
```  r->compression_ctx.type()```;
and
https://github.com/facebook/rocksdb/commit/01013ae7666af78d4b0fc1e1baf4a735a8db5344 changed it back from 
```  r->compression_ctx.type();```
to
```  r->compression_type;```
 
I will update the PR because of the change in master since the PR was made.